server/main.js

```diff
@@ -1,6 +1,7 @@
 const UDP_PORT = 47269;
 const CMD_UDP_PORT = 47268;
 const HTTP_WS_PORT = 8080;
+const requestDelay = 20;// every requestDelay milliseconds we send a udp packet 
```

Comment:
20ms corresponds to 50fps, we can probably reduce to 24fps (like some video) while keeping the a fluid rendering. (41ms)

Reply:
After testing, you can even go to the `50ms` round number as it's remains fluid-enough for 3D animations and actually improves the performances.

Reply:
Notice that the comment mentions udp packets while it actually sends websocket messages